demo/src/screens/componentScreens/CheckboxScreen.tsx

```diff
@@ -67,6 +70,32 @@ export default class CheckboxScreen extends Component {
             iconColor={Colors.green10}
           />
         </View>
+
+        <View marginT-20>
+          <Text text60 $textDefault marginB-10>
+            Validation
+          </Text>
+          <View row spread marginB-10>
+            <Checkbox
+              ref={this.checkbox}
+              value={this.state.value7}
+              onValueChange={value7 => this.setState({value7})}
+              label={'This is a checkbox'}
+            />
+            <Button size={'small'} label={'Validate'} onPress={() => this.checkbox.current?.validate()}/>
+          </View>
+          <View row spread>
+            <Checkbox
+              disabled
+              ref={this.checkbox1}
+              value={this.state.value7}
+              onValueChange={value7 => this.setState({value7})}
+              iconColor={Colors.green10}
+              label={'This is disabled checkbox'}
+            />
+            <Button disabled size={'small'} label={'Validate'} onPress={() => this.checkbox1.current?.validate()}/>
+          </View>
```

Comment:
What is this (second) example testing?

Reply:
That there is no validation on a disabled checkbox